frontend/app/layout.tsx

```diff
@@ -14,6 +14,7 @@ import { ThemeProvider } from "@mui/material/styles";
 import Link from "next/link";
 import React from "react";
 import theme from "./theme";
+import AppHeader from "./AppHeader";
```

Comment:
Where is this file? Looks like you are importing a non existent file?

Reply:
Pls check latest commit